blueocean-dashboard/src/main/js/components/InputStep.jsx

```diff
@@ -69,17 +70,17 @@ export default class InputStep extends Component {
      * Submit the form as "cancel" out of the state data id.
      */
     cancelForm() {
-        const { id } = this.state;
-        parameterApi.cancelInputParameter(this.state.href, id);
+        const { href, id } = this.state;
+        parameterApi.cancelInputParameter(href, id);
```

Comment:
An example of how I don't get some of these new ES6 "enhancements".

It's a puzzle to me how ...

```javascript
const { href, id } = this.state;
parameterApi.cancelInputParameter(href, id);
```

is an improvement on ....

```javascript
parameterApi.cancelInputParameter(this.state.href, this.state.id);
```

Just say'n 😈 and I know @scherler will call me an old fashioned fart and to 🔥 in hell 😄